default.nix

```diff
@@ -1,12 +1,15 @@
 let
-  requiredVersion = import ./lib/minver.nix;
+  missingFeatures = import ./lib/minfeatures.nix;
 in
 
-if !builtins ? nixVersion || builtins.compareVersions requiredVersion builtins.nixVersion == 1 then
+if builtins.length missingFeatures > 0 then
```

Comment:
IIUC, `missingFeatures != []` is slightly more efficient since it won't count the list's length.

Not important here where the list is always small, but it's a good habit to get into for other scenarios.